### ISSUE TITLE
To those whom cannot reproduce the results on own datasets

### ISSUE THREAD

**Issue by hubert0527** [afruehstueck/tileGAN#11]
***TL;DR***  The author has modified the PGGAN architecture in their project, so your model pretrained on the original PGGAN codebase cannot be used here. You can delete it now.

***Related Issue:***
https://github.com/afruehstueck/tileGAN/issues/10

***Observation***
Since the network architecture is altered, so it is expected to generate random pixels like this:
![image](https://user-images.githubusercontent.com/17472642/141414489-811d7d5e-acd4-4f02-9b4e-c780df27aa16.png)

You may wonder why the preview thumbnails are working as expected. It is because the preview thumbnails are generated with the graphs stored in your own checkpoint, while the canvas above is generated with an altered model network architecture that breaks your model. 

***Solution***
1. Replace [these lines](https://github.com/tkarras/progressive_growing_of_gans/blob/2504c3f3cb98ca58751610ad61fa1097313152bd/networks.py#L287-L308) with [these lines](https://github.com/afruehstueck/tileGAN/blob/0460e228b1109528a0fefc6569b970c2934a649d/networks.py#L267-L271)
2. Train the PGGAN model all over again.
3. Replace [this line](https://github.com/afruehstueck/tileGAN/blob/0460e228b1109528a0fefc6569b970c2934a649d/networks.py#L375) with the codes below. This avoids loading modules that do not exist.
```
if out_res < resolution_log2:
    images_out = tf.identity(x)
else:
    images_out = torgb(x, res)
```
4. You're good to go.
